src/omg-cli/omg/setup/project.go

```diff
@@ -38,7 +38,7 @@ type QuotaError struct {
 	Region string
 }
 
-var UnsatisfiedQuotaErr = errors.New("Compute Engine quota is unsatisfied, request an increase at: https://console.cloud.google.com/iam-admin/quotas")
+var ErrUnsatisfiedQuota = errors.New("unsatisfied quota for Compute Engine, request an increase at: https://console.cloud.google.com/iam-admin/quotas")
```

Comment:
I don't think Compute Engine should be in this string since the quotas are for project resources--even if they're technically part of the "compute engine" APIs

Reply:
updated